### PR TITLE
qemu/tests: Update multi host part for floppy.py

### PR DESCRIPTION
**Pull request by spcui**
Signed-off-by: Shuping Cui <scui@redhat.com>

ID: 1288975

### CODE REVIEW

File: qemu/tests/floppy.py [hunk 1]
```diff
@@ -138,8 +138,7 @@ def test(self):
             error.context("Copying file to the floppy")
             md5_cmd = params.get("md5_cmd")
```
Comment (CongSmile) [inline qemu/tests/floppy.py]: If the md5_cmd is not existed, does it need raise error here directly?

File: qemu/tests/floppy.py [hunk 2]
```diff
@@ -138,8 +138,7 @@ def test(self):
...
             if md5_cmd:
-                md5_source = self.session.cmd("%s %s" % (params["md5_cmd"],
-                                                         source_file))
+                md5_source = self.session.cmd("%s %s" % (md5_cmd, source_file))
```
Comment (CongSmile) [inline qemu/tests/floppy.py]: I think no need for the else branch here, maybe you can help update it also.
md5_source = None
if md5_cmd:
    md5_source = ****

File: qemu/tests/floppy.py (hunk 1 above)
Comment (spcui) [inline qemu/tests/floppy.py]: no, if not existed, md5_cmd = None

File: qemu/tests/floppy.py (hunk 2 above)
Comment (spcui) [inline qemu/tests/floppy.py]: not need update to it.

**Comment by luckyh** [comment]
Acked-by: @luckyh

File: qemu/tests/floppy.py [hunk 3]
```diff
@@ -193,7 +191,7 @@ def test(self):
             self.srchost = params["hosts"][0]
             self.dsthost = params["hosts"][1]
             self.is_src = params["hostid"] == self.srchost
-            self.mig = self.mig_type(test, params, env, False, )
+            self.mig = self.mig_type(test, params, env, True, )
```
Comment (xutian) [inline qemu/tests/floppy.py]: @spcui   prepare_env = True , will create image file before VM create,  if set it to true,  how to ensure old guest os image not be recreate ?

Comment (spcui) [inline qemu/tests/floppy.py]: True means "preprocess_env=True", just invoke storage.preprocess_images, not re-create.

Comment (xutian) [inline qemu/tests/floppy.py]: I know it, I mean when storage.preprocess_images called, how to ensure OS image file not be recreate?

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by suqinhuang** [comment]
Thanks @luckyh merged

**Pull request merged**